examples/with-react-md/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Example app with react-md
+
+## How to use
+
+Download the example [or clone the repo](https://github.com/zeit/next.js):
+
+```bash
+curl https://codeload.github.com/zeit/next.js/tar.gz/master | tar -xz --strip=2 next.js-master/examples/with-react-md
+cd with-react-md
+```
+
+Install it and run:
+
+```bash
+npm install
+npm run dev
+```
+
+Deploy it to the cloud with [now](https://zeit.co/now) ([download](https://zeit.co/download))
+
+```bash
+now
+```
+
+## The idea behind the example
+
+This example features how yo use [react-md](https://react-md.mlaursen.com/) (React Material Design) with Next.js.
+
+I recommend reading [layout-component](../layout-component) example next to learn how to reuse the layout across the pages.
```

Comment:
I just wanted to ask about use of the same header but with different titles and the example answered it, thanks